accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -0,0 +1,360 @@
+    - `OpenExisting(String, MemoryMappedFileRights, HandleInheritability)`
+
+### System.IO.Pipes
+
+* **NamedPipeClientStream**
+    - `get_NumberOfServerInstances()`
+* **PipeStream**
+    - `WaitForPipeDrain()`
+* **PipeTransmissionMode**
+    - `Message`
+
+### System.Net
+
+* **HttpListenerTimeoutManager**
+    - `set_EntityBody(TimeSpan)`
+    - `set_HeaderWait(TimeSpan)`
+    - `set_MinSendBytesPerSecond(Int64)`
+    - `set_RequestQueue(TimeSpan)`
+
+### System.Net.Sockets
+
+* **IOControlCode** (Enum -- _all members except `NonBlockingIO`, `DataToRead`, and `OobDataRead`_)
+    - `AsyncIO`
+    - `AssociateHandle`
+    - `EnableCircularQueuing`
+    - `Flush`
+    - `GetBroadcastAddress`
+    - `GetExtensionFunctionPointer`
+    - `GetQos`
+    - `GetGroupQos`
+    - `MultipointLoopback`
+    - `MulticastScope`
+    - `SetQos`
+    - `SetGroupQos`
+    - `TranslateHandle`
+    - `RoutingInterfaceQuery`
+    - `RoutingInterfaceChange`
+    - `AddressListQuery`
+    - `AddressListChange`
+    - `QueryTargetPnpHandle`
+    - `NamespaceChange`
+    - `AddressListSort`
+    - `ReceiveAll`
+    - `ReceiveAllMulticast`
+    - `ReceiveAllIgmpMulticast`
+    - `KeepAliveValues`
+    - `AbsorbRouterAlert`
+    - `UnicastInterface`
+    - `LimitBroadcasts`
+    - `BindToInterface`
+    - `MulticastInterface`
+    - `AddMulticastGroupOnInterface`
+    - `DeleteMulticastGroupFromInterface`
+* **Socket**
+    - `SetIPProtectionLevel(IPProtectionLevel)`
+* **TcpListener**
+    - `AllowNatTraversal(Boolean)`
+* **TransmitFileOptions**
+    - `UseKernelApc`
+    - `UseSystemThread`
+    - `WriteBehind`
+* **UdpClient**
+    - `AllowNatTraversal(Boolean)`
+
+### System.Runtime.InteropServices
+
+* **DispatchWrapper**
+    - `get_WrappedObject()`
+* **Marshal**
+    - `AddRef(IntPtr)`
+    - `BindToMoniker(String)`
+    - `ChangeWrapperHandleStrength(Object, Boolean)`
+    - `CreateAggregatedObject(IntPtr, Object)`
+    - `CreateAggregatedObject<T>(IntPtr, T)`
+    - `FinalReleaseComObject(Object)`
+    - `GetComInterfaceForObject(Object, Type)`
+    - `GetComInterfaceForObject(Object, Type, CustomQueryInterfaceMode)`
+    - `GetComInterfaceForObject<T, TInterface>(T)`
+    - `GetIUnknownForObject(Object)`
+    - `GetNativeVariantForObject(Object, IntPtr)`
+    - `GetNativeVariantForObject<T>(T, IntPtr)`
+    - `GetObjectForIUnknown(IntPtr)`
+    - `GetObjectForNativeVariant(IntPtr)`
+    - `GetObjectForNativeVariant<T>(IntPtr)`
+    - `GetObjectsForNativeVariants(IntPtr, Int32)`
+    - `GetObjectsForNativeVariants<T>(IntPtr, Int32)`
+    - `GetStartComSlot(Type)`
+    - `GetTypedObjectForIUnknown(IntPtr, Type)`
+    - `GetTypeFromCLSID(Guid)`
+    - `GetTypeInfoName(ITypeInfo)`
+    - `GetUniqueObjectForIUnknown(IntPtr)`
+    - `QueryInterface(IntPtr, Guid, IntPtr)`
+    - `Release(IntPtr)`
+    - `ReleaseComObject(Object)`
+
+### System.Security.Cryptography
+
+* **CspKeyContainerInfo**
+    - `get_Accessible()`
```

Comment:
`ctor` is missing: https://github.com/dotnet/runtime/blob/ecdf9de474459f17efb90fd64609d164bcf454d4/src/libraries/System.Security.Cryptography.Csp/src/System/Security/Cryptography/CspKeyContainerInfo.Unix.cs#L8

```suggestion
    - `ctor()`
    - `get_Accessible()`
```